core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java

```diff
@@ -309,65 +304,39 @@ protected enum CommitStatus {
    * @return Commit Status of Success, Failure or Unknown
    */
   protected CommitStatus checkCommitStatus(String newMetadataLocation, TableMetadata config) {
-    int maxAttempts =
```

Comment:
Since Hive-View also needs to check the commit status, so it has been moved to an abstract class `BaseMetastoreOperations`.